tools/yaml-templates/android-test.yml

```diff
@@ -52,4 +51,4 @@ steps:
       testRunTitle: 'E2E Tests - Android'
       searchFolder: '$(Agent.BuildDirectory)/androidHost'
       mergeTestResults: true
-    condition: succeededOrFailed()
+    condition: or(succeededOrFailed(), eq(variables['Agent.JobStatus'], 'Canceled'))
```

Comment:
Is the change gonna publish result anyway even if E2E task is cancelled?

Reply:
Yes, that's the goal, succeededOrFailed() resolves to false when a task is cancelled due to timeout

Reply:
> Yes, that's the goal, succeededOrFailed() resolves to false when a task is cancelled due to timeout

Great. Another lesson learned as iOS could do the same thing leveraging on what you found. I would do some tests later.